dev/Makefile

```diff
@@ -2,34 +2,37 @@ EXPECTED_CONTEXT=colima
 NAMESPACE=optimus-dev
 OPTIMUS_ODPF_CHART=odpf/optimus
 
+DAGS_PATH?=./dags
+OPTIMUS_PLUGINS_PATH?=./plugins
```

Comment:
Here the assumption is to plugins are copied to this folder (extra step)

Reply:
remove this one, use artifacts list inside setup.yaml instead